data/json/npcs/BGSS_talk_common.json

```diff
@@ -0,0 +1,704 @@
+        "trial": { "type": "INTIMIDATE", "difficulty": 50, "mod": [ [ "BRAVERY", 1 ] ] },
+        "success": { "topic": "BGSS_LOST_PARTNER_2_STORY2" },
+        "failure": { "topic": "BGSS_LOST_PARTNER_2_FUCKOFF" }
+      },
+
+     { "text": "Never mind. Sorry I brought it up.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_2_NOTYET",
+    "type": "talk_topic",
+    "dynamic_line": "I appreciate the sentiment, but I don't think it would. Drop it.",
+    "responses": [
+     { "text": "OK.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_2_FUCKOFF",
+    "type": "talk_topic",
+    "dynamic_line": "Fuck off. This doesn't have anything to do with you, or with us.",
+    "responses": [
+     { "text": "...", "topic": "TALK_DONE" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_2_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "All right, fine. I had someone. I lost them.",
+    "responses": [
+    { "text": "What happened?", "topic": "BGSS_LOST_PARTNER_2_STORY3" },
+    { "text": "Never mind. Sorry I brought it up.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_2_STORY3",
+    "type": "talk_topic",
+    "dynamic_line": "They weres at home when the bombs started dropping and the world went to hell. I was at work. I tried to make it to our house, but the city was a war zone. Things I can't describe lurching through the streets, crushing people and cars. Soldiers trying to stop them, but hitting people in the crossfire as much as anything. And then the collateral damage would get right back up and join the enemy. If it hadn't been for my partner, I would have just left, but I did what I could and I slipped through. I actually fucking made it alive.",
+    "responses": [
+    { "text": "You must have seen some shit.", "topic": "BGSS_LOST_PARTNER_2_SOMESHIT" },
+    { "text": "I take it home was bad.", "topic": "BGSS_LOST_PARTNER_2_HOME" },
+    { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_2_SOMESHIT",
+    "type": "talk_topic",
+    "dynamic_line": "Yeah. I really did. It took me two days to make it across the city on foot, camping out in dumpsters and places like that. I started moving more by night, and I learned right away to avoid the military. They were a magnet for the dead, and they were usually stationed where the monsters were coming. Some parts of the city were pretty tame at first. There were a few chunks where people had been evacuated or cleared out, and the monsters didn't really go there. Later on, others like me started moving into those neighbourhoods, so I switched and started running through the blasted out downtown. I had to anyway, to get home. By the time I made the switch though, the fighting was starting to die off, and I was mostly just avoiding attention from zombies and other things.",
```

Comment:
```suggestion
    "dynamic_line": "Yeah. I really did. It took me two days to make it across the city on foot, camping out in dumpsters and places like that. I started moving more by night, and I learned right away to avoid the military. They were a magnet for the dead, and they were usually stationed where the monsters were coming. Some parts of the city were pretty tame at first. There were a few chunks where people had been evacuated or cleared out, and the monsters didn't really go there. Later on, others like me started moving into those neighborhoods, so I switched and started running through the blasted out downtown. I had to anyway, to get home. By the time I made the switch though, the fighting was starting to die off, and I was mostly just avoiding attention from zombies and other things.",
```